napari/layers/image/image.py

```diff
@@ -314,6 +360,8 @@ def rendering(self, rendering):
 
     def _set_view_slice(self):
         """Set the view given the indices to slice with."""
+        nd = self.dims.not_displayed
```

Comment:
I don't like this abbreviation, as it usually means "n-dimensional"

Reply:
replaced with `not_disp`